CONTRIBUTING.md

```diff
@@ -489,6 +490,35 @@ Your PR is now updated on GitHub. To ensure team member are aware of this,
 leave a message on the PR stating something like, "review feedback applied".
 Then, the team is notified and able to re-review your PR more quickly.
 
+### Temporarily blocking a PR
+
+Sometimes a submitter or reviewer of a PR may need to temporarily block
+a PR from being merged. There can be a number of reasons for this, including:
+
+- The PR has been opened as a Request For Comments (RFC).
+- The PR is incomplete, needs more work or rework.
+- The PR is a Work In Progress (WIP), which has been opened
```

Comment:
Isn't this the same as the previous bullet? If not, could you add an appropriate label in brackets for the previous bullet?

Reply:
In my mind nominally not the same - one is you know it needs more work, the other is an 'early preview'. An RFC is more like 'I think it is done, what do you think'. I've added 'do-not-merge' to the middle one now.

Reply:
Looking at this again, I think it would be clearer to combine the bullets and the table. Something like the following, along with a note along the lines of, "either apply the github label or update the PR description summary" to flag your particular scenario".

| Scenario | github label | PR description contains |
|-|-|-|
| PR created "as an idea" and feedback wanted | `rfc` | RFC |
| PR incomplete - needs more work or rework | `do-not-merge` | DNM |
| PR should not be merged (has all required "acks", but needs more reviewer input) | `do-not-merge` | DNM |
| PR is a "work In progress", raised to get early feedback | `wip` | WIP |
| PR is complete but depends on another so should not be merged (yet) | `do-not-merge` | DNM |